config/sync/field.field.node.recipe.field_r_cost.yml

```diff
@@ -11,7 +11,7 @@ entity_type: node
 bundle: recipe
 label: Cost
 description: ''
-required: true
```

Comment:
How come you've changed this?

Reply:
Cost is not always easy to calculate when making a recipe. It makes sense for a site displaying healthy recipes for people but when individuals add their own recipes, the cost field might put people off contributing. I know it would put me off personally, if I had to include it.

Reply:
Fair plays 👍